README.md

```diff
@@ -3,6 +3,8 @@
 This README would normally document whatever steps are necessary to get the
 application up and running.
 
+
```

Comment:
- [ ] The backend project still needs a proper readme file.  Kindly create one as instructed in the last review. 👍 

- [ ] Also, kindly delete the "test" folder if not using it as mentioned in the last review.

Reply:
Thank you!